tools/cover-by-test.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+
+set -x
+
+for testfile in $(find t -name "*.t"); do
```

Comment:
I am not sure if we follow shell check for our shell scripts, but using find like this can be problematic if find encounters whitespace or glob patterns in filename, if we do not allow whitespace or special characters in file names then it is fine
Refer: https://www.shellcheck.net/wiki/SC2044

Reply:
I guess it doesn't matter here but I'd also like it this wouldn't break on white-spaces.